frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/expectedkeys/IExpectsKeyProject.java

```diff
@@ -26,7 +26,7 @@ default IBranchLayoutWriter getBranchLayoutWriter(AnActionEvent anActionEvent) {
     return getProject(anActionEvent).getService(BranchLayoutWriterProvider.class).getBranchLayoutWriter();
   }
 
-  default BaseGraphTable getGraphTable(AnActionEvent anActionEvent) {
+  default AbstractEnhancedGraphTable getGraphTable(AnActionEvent anActionEvent) {
```

Comment:
Hmm we don't use `Abstract` prefix in principle, rather `Base`... any specific reason here?

Reply:
Just because `BaseEnhancedGraphTable` sounds weird (for me) and it can sounds similar to `BaseGraphTable`

Reply:
Finally renamed to `BaseEnhancedGraphTable`